--- drgeo-1.0.0/configure.ac~	2005-06-12 01:16:31.311833264 +0000
+++ drgeo-1.0.0/configure.ac	2005-06-12 01:19:01.399016544 +0000
@@ -53,7 +53,7 @@
 dnl **********************************************************************
 dnl * 	INTERNATIONALIZATION SUPPORT					 *
 dnl **********************************************************************
-ALL_LINGUAS="az ca cs da de el en_CA en_GB es eu fr ga gl gu hi hr hu it ja ml mr nb ne nl no pl pt pt_BR ru rw sl sq sr sr@Latn sv uk zh_CN zh_TW"
+ALL_LINGUAS="az ca cs da de el en_CA en_GB es eu fr ga gl gu hi hr hu it ja ml mr nb ne nl    pl pt pt_BR ru rw sl sq sr sr@Latn sv uk zh_CN zh_TW"
 
 GETTEXT_PACKAGE=drgeo
 AC_SUBST(GETTEXT_PACKAGE)
